src/guides/v2.3/extension-dev-guide/extend-catalog-rule-conditions.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Extend Catalog Rule Conditions
+contributor_name: Ziffity
+contributor_link: https://www.ziffity.com/
+group: php-developer-guide
+---
+
+Catalog price rules can be used to offer products to buyers at a discounted price, based on a set of defined conditions. Catalog price rules do not use coupon codes. The discounts are applied to the final product price in the product listing and the product description page even before adding the product to the shopping cart.
+
+{:.bs-callout-info}
+For more information about catalog price rules, refer to [Catalog Rules](https://docs.magento.com/user-guide/marketing/price-rules-catalog.html).
+
+## Default Conditions
+
+In the `Conditions` tab, by default, Magento comes with the following product attributes in the `add conditions` section.
+
+![Catalog Rule Conditions]({{ site.baseurl }}/common/images/conditions-section-v2.png)
+
+## Implementation of Existing Conditions
+
+In the `app/code/Magento/CatalogRule/etc/di.xml` configuration, the following `type` configuration is defined:
+
+```xml
+<type name="Magento\CatalogRule\Model\Rule\Condition\Combine">
+    <arguments>
+        <argument name="data" xsi:type="array">
+            <item name="form_name" xsi:type="string">catalog_rule_form</item>
+        </argument>
+    </arguments>
+</type>
+<type name="Magento\CatalogRule\Model\Rule\Condition\Product">
+    <arguments>
+        <argument name="data" xsi:type="array">
+            <item name="form_name" xsi:type="string">catalog_rule_form</item>
+        </argument>
+    </arguments>
+</type>
+```
+
+In the above configuration, the `catalog_rule_form` is a UI component form. It is defined in the `app/code/Magento/CatalogRule/view/adminhtml/ui_component/catalog_rule_form.xml` file.
+
+The `Magento\CatalogRule\Model\Rule\Condition\Combine` and the `Magento\CatalogRule\Model\Rule\Condition\Product` classes are important for the listed conditions and validations of the specified conditions.
```

Comment:
Why are these important?

Reply:
I rewrote it. Please confirm if it is correct.

Reply:
@dobooth Yep, It is correct.